votes/0675-Oreoxmt-as-docs-committer.md

```diff
@@ -0,0 +1,35 @@
+# A Vote for Oreoxmt as Docs Reviewer
+
+## Proposal
+
+[@Oreoxmt](https://github.com/Oreoxmt) has been working on TiDB documentation for the past three months. Her work focused on translating TiDB documents from Chinese to English or from English to Chinese, managing documentation projects, troubleshooting docs CI, and conducting docs usability tests. She has delivered documentation for [TiDB Developer Guide](https://docs.pingcap.com/tidb/stable/dev-guide-overview) and [PingCAP Clinic](https://docs.pingcap.com/tidb/dev/quick-start-with-clinic), and helped identifying slow or failed CI tests and managed to improve them.
+
+I think @Oreoxmt shows her capability and commitment as a committer of the Docs Team. The details of her contributions are listed as follows:
+
+- Authored pull requests
+    - [docs](https://github.com/pingcap/docs/pulls?q=is%3Apr+is%3Amerged+author%3AOreoxmt): 57
+    - [docs-cn](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+is%3Amerged+author%3AOreoxmt): 44
+    - [docs-tidb-operator](https://github.com/pingcap/docs-tidb-operator/pulls?q=is:pr+is:merged+author:Oreoxmt): 10
+    - [tiflash](https://github.com/pingcap/tiflash/pulls?q=is:pr+is:merged+author:Oreoxmt): 1
+- Reviewed pull requests
+    - [docs](https://github.com/pingcap/docs/pulls?q=is%3Apr+reviewed-by%3AOreoxmt+is%3Amerged+-author%3Ati-chi-bot+): 78
+    - [docs-cn](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+reviewed-by%3AOreoxmt+is%3Amerged+-author%3Ati-chi-bot+): 64
+    - [docs-tidb-operator](https://github.com/pingcap/docs-tidb-operator/pulls?q=is%3Apr+reviewed-by%3AOreoxmt+is%3Amerged+-author%3Ati-chi-bot+): 8
+- Resolved issues
+    - [docs](https://github.com/pingcap/docs/issues?q=is%3Aissue+is%3Aclosed+assignee%3AOreoxmt): 1
+    - [docs-cn](https://github.com/pingcap/docs-cn/issues?q=is%3Aissue+is%3Aclosed+assignee%3AOreoxmt): 4
+    - [docs-dm](https://github.com/pingcap/docs-dm/issues?q=is:issue+is:closed+assignee:Oreoxmt): 3
+    - [docs-tidb-operator](https://github.com/pingcap/docs-tidb-operator/issues?q=is%3Aissue+is%3Aclosed+assignee%3AOreoxmt): 1
+
+I (@TomShawn) hereby nominate @Oreoxmt as Docs Committer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team: Docs
+
+## Result
+
```

Comment:
@TomShawn Please help update the voting result here.

Reply:
Done